.github/auto-comment.yml

```diff
@@ -0,0 +1,10 @@
+pullRequestOpened: >
+  Thank you for raising your pull request. Please note that the main development of Kata Containers
+  has moved to the 2.0-dev branch of https://github.com/kata-containers/kata-containers repository.
```

Comment:
Might be worth providing a direct link to the 2.0 dev branch:

> has moved to the [`2.0-dev`](https://github.com/kata-containers/kata-containers/tree/2.0-dev) branch of ...

Reply:
Good point! Fixed.